src/stories/DomainSlider.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { CustomDomain, DomainSlider, ScaleType } from '../packages/lib';
+import type { DomainSliderProps } from '../h5web/toolbar/controls/DomainSlider/DomainSlider';
+import { useState } from 'react';
+import FlexCenter from './decorators/FlexCenter';
+
+const Template: Story<
+  Omit<DomainSliderProps, 'customDomain' | 'onCustomDomainChange'>
+> = (args) => {
+  const [customDomain, setCustomDomain] = useState<CustomDomain>([null, null]);
+
+  return (
+    <DomainSlider
+      customDomain={customDomain}
+      onCustomDomainChange={setCustomDomain}
+      {...args}
+    />
+  );
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  dataDomain: [-95, 400],
+  scaleType: ScaleType.Linear,
+};
+
+export const Histogram = Template.bind({});
+
+Histogram.args = {
+  ...Default.args,
+  histogram: {
+    data: [100, 166, 130, 92, 76, 68, 60, 52, 50, 26],
+    bins: [-95, -45.5, 4, 53.5, 103, 152.5, 202, 251.5, 301, 350.5, 400],
+  },
+};
+
+export default {
+  title: 'Building Blocks/DomainSlider',
+  component: DomainSlider,
+  decorators: [FlexCenter],
+  parameters: {
+    controls: { exclude: ['customDomain', 'onCustomDomainChange'] },
+  },
+  argTypes: {
+    scaleType: {
+      control: {
+        type: 'inline-radio',
+        options: [ScaleType.Linear, ScaleType.Log, ScaleType.SymLog],
+      },
+    },
+  },
+} as Meta;
```

Comment:
Even if I think that `DomainSlider` does not need a story of its own, I found way it easier to test the histogram with it